demisto_sdk/commands/generate_outputs/json_to_outputs/json_to_outputs.py

```diff
@@ -99,10 +99,10 @@ def flatten(x, name=""):
         except IndexError:
             name = name.title() if camelize else name
 
-        if isinstance(x, dict):
+        if isinstance(x, dict) and x:
             for a in x:
                 flatten(x[a], name + a + ".")
```

Comment:
instead of adding this condition, can we edit `flatten` to accept an empty input?

Reply:
`flatten` accepts an empty input

Reply:
I think the `and x` here can be pulled upwards, OR we can change the following line to `for a in x or ()` :)